### ISSUE TITLE
Usb error on start

### ISSUE THREAD

**Issue by spotUP** [GXTX/YACardEmu#4]
Hello, i get this error when i start it:

C:\Users\User\Desktop\cardemu>YACardEmu.exe
[critical] SerIo::Init: Failed to open /dev/ttyUSB1
[critical] Couldn't initalize the serial controller.'

am i missing something?

**Comment by GXTX** [comment]
Yes, you using Windows and trying to run with a Linux device path. Windows only works with `COMn` and pipes.

**Comment by spotUP** [comment]
Can you paste a working config example?

**Comment by GXTX** [comment]
I don't know what kind of setup you're using. Look at the sample config that comes with the program, it has all the options.
